test/TestProviderRound.spec.js

```diff
@@ -98,6 +98,41 @@ contract('ProviderRound', accounts => {
 
   });
 
+  describe('resignAsProvider', () => {
+
+    before(async () => {
+      providerRound = await ProviderRound.new();
+      await blockMiner.mineUntilEndOfElectionPeriod(providerRound);
+      await providerRound.initializeRound();
+    });
+
+    it('should remove a provider from the provider mapping', async () => {
+      await providerRound.provider(22, 10, 1, 25, {from: accounts[0]});
+      const registeredProvider = await providerRound.providers.call(accounts[0]);
+      assert.equal(PROVIDER_REGISTERED, registeredProvider[0])
```

Comment:
Can you call `registeredProvider.status` instead of `registeredProvider[0]` here?

Reply:
Unfortunately web3js returns a list instead of an object when calling a Solidity Struct.
I thought it wasn't worth converting to a JS object with the right field